chapter_notation/index.md

```diff
@@ -1,85 +1,94 @@
 # Notation
 :label:`chap_notation`
 
-The notation used throughout this book is summarized below.
+Throughout this book, we adhere to the following notatinoal conventions.
+Note that some of these symbols are placeholders,
+while others refer to specific objects.
+As a general rule of thumb, 
+the indefinite article "a" indicates
+that the symbol is a placeholder
+and that similarly formatted symbols
+can denote other objects of the same type.
+For example, "$x$: a scalar" means 
+that lowercased letters generally
+represent scalar values.
 
 
-## Numbers
 
-* $x$: A scalar
-* $\mathbf{x}$: A vector
-* $\mathbf{X}$: A matrix
-* $\mathsf{X}$: A tensor
-* $\mathbf{I}$: An identity matrix
-* $x_i$, $[\mathbf{x}]_i$: The $i^\mathrm{th}$ element of vector $\mathbf{x}$
-* $x_{ij}$, $x_{i,j}$,$[\mathbf{X}]_{ij}$, $[\mathbf{X}]_{i,j}$: The element of matrix $\mathbf{X}$ at row $i$ and column $j$
+## Numerical Objects
+
+* $x$: a scalar
+* $\mathbf{x}$: a vector
+* $\mathbf{X}$: a matrix
+* $\mathsf{X}$: a general tensor
+* $\mathbf{I}$: an identity matrix---square, with $1$ on all diagonal entries and $0$ on all off-diagonals
+* $x_i$, $[\mathbf{x}]_i$: the $i^\mathrm{th}$ element of vector $\mathbf{x}$
+* $x_{ij}$, $x_{i,j}$,$[\mathbf{X}]_{ij}$, $[\mathbf{X}]_{i,j}$: the element of matrix $\mathbf{X}$ at row $i$ and column $j$.
 
 
 
 
 ## Set Theory
 
 
-* $\mathcal{X}$: A set
-* $\mathbb{Z}$: The set of integers
-* $\mathbb{Z}^+$: The set of positive integers
-* $\mathbb{R}$: The set of real numbers
-* $\mathbb{R}^n$: The set of $n$-dimensional vectors of real numbers
+* $\mathcal{X}$: a set
+* $\mathbb{Z}$: the set of integers
+* $\mathbb{Z}^+$: the set of positive integers
+* $\mathbb{R}$: the set of real numbers
+* $\mathbb{R}^n$: the set of $n$-dimensional vectors of real numbers
 * $\mathbb{R}^{a\times b}$: The set of matrices of real numbers with $a$ rows and $b$ columns
-* $|\mathcal{X}|$: Cardinality (number of elements) of set $\mathcal{X}$
-* $\mathcal{A}\cup\mathcal{B}$: Union of sets $\mathcal{A}$ and $\mathcal{B}$
-* $\mathcal{A}\cap\mathcal{B}$: Intersection of sets $\mathcal{A}$ and $\mathcal{B}$
-* $\mathcal{A}\setminus\mathcal{B}$: Subtraction of set $\mathcal{B}$ from set $\mathcal{A}$
+* $|\mathcal{X}|$: cardinality (number of elements) of set $\mathcal{X}$
+* $\mathcal{A}\cup\mathcal{B}$: union of sets $\mathcal{A}$ and $\mathcal{B}$
+* $\mathcal{A}\cap\mathcal{B}$: intersection of sets $\mathcal{A}$ and $\mathcal{B}$
+* $\mathcal{A}\setminus\mathcal{B}$: set subtraction of $\mathcal{B}$ from $\mathcal{A}$ (contains only those elements of $\mathcal{A} that do not belong to $\mathcal{B}$)
```

Comment:
Are we missing a "$" after $\mathcal{A}?

Reply:
fixed